### PR TITLE
[UPDATE] nostrnip5 to v0.1.1

### PR DESCRIPTION
**Pull request by alanbits**
https://github.com/lnbits/nostrnip5/releases/v0.1.1

### CODE REVIEW

**Review by dni — APPROVED** [review]

**Review by motorina0 — APPROVED** [review]

File: extensions.json [hunk 1]
```diff
@@ -383,13 +383,13 @@
...
-            "version": "0.1.0",
+            "version": "0.1.1",
             "min_lnbits_version": "0.12.5",
             "short_description": "Create NIP05 address for your domains",
             "icon": "https://github.com/lnbits/nostrnip5/raw/main/static/image/nostrnip5.png",
             "details_link": "https://raw.githubusercontent.com/lnbits/nostrnip5/main/config.json",
-            "archive": "https://github.com/lnbits/nostrnip5/archive/refs/tags/v0.1.0.zip",
-            "hash": "fe9f1e6482ceed6381628d144ff2c66b696cb90d0f5a782f544537f0a5b28062"
+            "archive": "https://github.com/lnbits/nostrnip5/archive/refs/tags/v0.1.1.zip",
+            "hash": "10d56f3fd6754f9eadae71e14232d143ca7116fba8a05fab87dabc2ee5ba711c",
```
Comment (motorina0) [inline extensions.json:392]: @alanbits why did you add the extra comma?

Comment (dni) [reply extensions.json:392]: :thinking: why actually and the first time?

Comment (dni) [reply extensions.json:392]: i think it was me :D with the merge

Comment (dni) [reply extensions.json]: fixed

**Review by arcbtc — APPROVED** [review]
